### ISSUE TITLE
[BUG] I am getting signed out frequently even though I am signed in on my AWS credentials file

### ISSUE THREAD

**Issue by niyazpk** [aws-samples/amazon-bedrock-client-for-mac#38]
**Describe the bug**
Pretty sure this used to work without issues before, but in the latest release it does not work anymore.

tldr: I am signed in using `aws configure` in my credentials file, and I am updating the credentials before expiry - but the app is signing me out frequently and gives a `token expired` error. only restarting the app fixes the issue

**To Reproduce**
1. `aws configure` (with refresh option set up locally).
2. Open the Bedrock app, and start using it.
3. After XX hours/minutes I start getting the error: 
```
Error invoking the model: UnknownAWSHTTPServiceError(typeName: Optional("ExpiredTokenException"), message: Optional("The security token included in the request is expired"), requestID: nil, requestID2: nil, httpResponse: Status Code: http_status_403 Date: Fri, 05 Jul 2024 19:31:02 GMT, Content-Type: application/json, Content-Length: 67, x-amzn-requestid: abefeee8-7400-4237-bb1f-8b5992d2696d, x-amzn-errortype: ExpiredTokenException:http://internal.amazon.com/coral/com.amazon.coral.service/)
```
4. Verified that I am still logged in on my AWS account and I can make the CLI calls without issue.

**Expected behavior**
I would like to continue using the app without restarting it.

**Environment (required):**
 - macOS version: 14.4.1
 - Amazon Bedrock Client for Mac version: 1.2.0

**Additional context (optional)**
If this already fixed in the latest commit, please cut a release so that we can use the update. Thanks!

**Comment by didhd** [comment]
Fixed in v1.2.1
